test-data/yul-sample/SimpleStore.yul

```diff
@@ -1,9 +1,9 @@
 object "SimpleStore" {
```

Comment:
Can you add this example to `test-data/zkync` instead of overriding the upstream one?

Reply:
Sure